sci-electronics/xcircuit/xcircuit-3.9.72.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools multilib
+
+DESCRIPTION="Circuit drawing and schematic capture program"
+SRC_URI="http://opencircuitdesign.com/xcircuit/archive/${P}.tgz"
+HOMEPAGE="http://opencircuitdesign.com/xcircuit"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+DEPEND="
+	app-text/ghostscript-gpl
+	dev-lang/tk:0
+	media-libs/fontconfig:1.0
+	sys-libs/zlib
+	x11-libs/cairo
+	x11-libs/libX11
+	x11-libs/libXt
+	x11-libs/libXpm
+	x11-libs/libSM
+	x11-libs/libICE
+"
+RDEPEND=${DEPEND}
+
+src_prepare() {
+	mv configure.{in,ac} || die
+	eautoreconf
+	default
+}
+
+src_configure() {
+	local myeconfargs=(
```

Comment:
you define this, then throw it away. myeconfargs is no magic variable, you'll have to run `econf "${myeconfargs[@]}"` instead of default below.